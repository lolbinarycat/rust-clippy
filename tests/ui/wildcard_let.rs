#![forbid(clippy::wildcard_let)]

fn main() {
    let _ = 1;
}
