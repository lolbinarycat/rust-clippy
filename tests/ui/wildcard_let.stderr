error: wildcard let
  --> tests/ui/wildcard_let.rs:4:9
   |
LL |     let _ = 1;
   |         ^
   |
   = help: remove this binding or handle the value
note: the lint level is defined here
  --> tests/ui/wildcard_let.rs:1:11
   |
LL | #![forbid(clippy::wildcard_let)]
   |           ^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

